utilities/blob_db/blob_db_impl.cc

```diff
@@ -1813,16 +1797,7 @@ Status BlobDBImpl::GCFileAndUpdateLSM(const std::shared_ptr<BlobFile>& bfptr,
       // because
       // a new version of the key came in at this time, which will override
       // the current version being iterated on.
-      if (s1.IsBusy()) {
-        ROCKS_LOG_INFO(db_options_.info_log,
-                       "Optimistic transaction failed delete: %s bn: %" PRIu32,
-                       bfptr->PathName().c_str(), gcstats->blob_count);
```

Comment:
The transaction will fail only when a newer version of the key is being insert, in which case we don't need to delete the key from LSM. So this is not an error case. Logging the info per key is just too noisy. I'm going to add a counter and report it through GCStats.